web/web_test.go

```diff
@@ -254,9 +251,8 @@ func TestCheckClientCompatability(t *testing.T) {
 	}
 	for _, browser := range uaTestParameters {
 		t.Run(browser.Name, func(t *testing.T) {
-			if result := CheckClientCompatability(browser.UserAgent); result != browser.Result {
-				t.Fatalf("%s User Agent Test failed!", browser.Name)
-			}
+		  result := CheckClientCompatability(browser.UserAgent)
```

Comment:
Thanks for re-adding that. Unfortunately, that messed up the indentation here by mixing spaces and tabs. Would you be able to fix that as well?
```suggestion
			result := CheckClientCompatability(browser.UserAgent)
```